app/crud/base.py

```diff
@@ -36,7 +36,7 @@ def get(self, db: Session, id: Any) -> Optional[ModelType]:
 
         Returns
         -------
-        Optional[ModelType]
+        Optional
```

Comment:
This was correct before (`Optional[ModelType]`). Same thing for the new changes in this section.

Reply:
Let me correct it and I'll make a new commit.